src/components/policy/policy_regular/src/policy_manager_impl.cc

```diff
@@ -58,7 +58,8 @@ void DeleteManager(policy::PolicyManager* pm) {
 }
 
 namespace {
-const uint32_t kDefaultRetryTimeoutInMSec = 60u * date_time::DateTime::MILLISECONDS_IN_SECOND;
+const uint32_t kDefaultRetryTimeoutInMSec =
```

Comment:
@LuxoftAKutsan 
This value is available from policy table, so this variable is not needed.